package.json

```diff
@@ -11,23 +11,15 @@
     "@nomiclabs/hardhat-web3": "^2.0.0",
     "@openzeppelin/contracts": "^5.0.1",
     "@openzeppelin/contracts-upgradeable": "^5.0.1",
-    "arb-ts": "^1.0.2",
     "chai": "^4.3.4",
     "dotenv": "^10.0.0",
```

Comment:
Why some of the packages such as prettier is removed?

Reply:
Just some cleanup work without breaking tests, plus we agreed to use forge as the formatter. Maybe it's more safe for everyone to revert it back.

Reply:
We can change to prettier to forge in prettier:check & prettier:fix of the package.json